.github/workflows/scripts/spam-detection/process-issue.sh

```diff
@@ -27,6 +28,17 @@ fi
 
 echo "detected as spam: $_issue_url"
 
-gh issue edit --add-label "$_suspected_spam_label" "$_issue_url"
+cat << EOF | gh issue comment "$_issue_url" --body-file -
+Thank you for taking the time to create this issue.
+
+We've automatically reviewed this issue and suspect it as potentially inauthentic or spam-like content. As a result, we're closing this issue.
+
+**If this was closed by mistake**, please don't hesitate to reach out to us by commenting on this issue with additional context.
+
+We appreciate your understanding and apologize if this action was taken in error. Our automated systems help us manage the large volume of issues we receive, but we know they're not perfect.
+EOF
+
+gh issue edit --add-label "suspected-spam" --add-label "invalid" "$_issue_url"
+gh issue close "$_issue_url"
```

Comment:
We should close the issue as "not planned", like how we're doing it:


```suggestion
gh issue close --reason 'not planned' "$_issue_url"
```